src/blocks/small-search.css

```diff
@@ -0,0 +1,45 @@
+.small-search__input {
+  font-weight: 400;
+  font-size: 14px;
+  margin: 0 auto;
+  width: 100%;
+  text-indent: 33px;
+  border: none;
+  background-image: url(../images/Search.svg);
+  background-repeat: no-repeat;
+  background-size: 11;
+  background-position: 13 14;
+}
+
+.main__small-search {
+  display: flex;
+  box-sizing: border-box;
+  margin: 20px 8px 0;
+  height: 42px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px 0 #3d4e611a;
+}
+
+.small-search__input:hover {
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 3px 8px 0 #3d4e6133;
+  background-image: url(../images/Search.svg);
+  background-repeat: no-repeat;
+  background-size: 11px;
+  background-position: 13px 14px;
+}
+
+.small-search__input:focus {
+  outline: none;
+  background: linear-gradient(180deg, #fff 100%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+  background-image: url(../images/Search.svg);
+  background-repeat: no-repeat;
+  background-size: 11px;
+  background-position: 13px 14px;
```

Comment:
The focus state for `.small-search__input` repeats the background image properties that are already defined in the default state. Consider removing these redundant styles to simplify the CSS.